nim/game.py

```diff
@@ -91,7 +97,18 @@ def __init__(self):
         self.greedy = Greedy()
         self.chaos = Chaos()
 
-    def new_game(self, min, max, piles):
```

Comment:
That's a good point.
Let's change:
```
    def new_game(self, min=DEFAULT_MIN, max=DEFAULT_MAX, piles=DEFAULT_PILES):
```
to
```
    @staticmethod
    def new_game(min=DEFAULT_MIN, max=DEFAULT_MAX, piles=DEFAULT_PILES):
```

Reply:
Can you give me a brief explanation on why we would do it this way? 
So you're using a decorator to make it a static function.. 
I'm guessing by 'method could be function' is that it could be a static global function. 
Does using the decorator instead of moving it out of here make a difference?

Reply:
> Does using the decorator instead of moving it out of here make a difference?

Technically: no.
But since we're creating a new game, it sort of  makes sense to have it in the `Game` class.
Since we're not using `self` in that method, I added the `@staticmethod` to let everyone know that yes it could be a function, but we're keeping it as a method for now and we're ok with it.

I wouldn't worry too much about this because this code will most likely need to change soon anyway. I mostly added this comment for 2 reasons:
1. To make @codacy-bot happy (you're welcome)
2. To introduce you to `@staticmethod`

Reply:
Soooo @staticmethod doesn't actually do anything other than make you write it so that it *could* be a static function? (As in why you have to remove self)

Reply:
I don't fully understand your question.
Maybe this will help? https://docs.python.org/3.6/library/functions.html#staticmethod